kansha/column/comp.py

```diff
@@ -73,7 +73,7 @@ def copy(self, parent, additional_data):
 
     def index_cards(self, cards):
         for card in cards:
-            scard = fts_schema.Card.from_model(card.data)
+            scard = fts_schema.Card.from_model(card)
             self.search_engine.add_document(scard)
```

Comment:
`self.search_engine.add(o)` -> the search engine accept any object `o` with a `to_document()` method